app/components/molecules/user-sanction-card.tsx

```
@@ -0,0 +1,162 @@
+import { ScaleIcon } from "lucide-react";
+
+export default function UserSanctionCard({
+  sanction,
+}: {
+  sanction: ParticipantSanction;
+}) {
+  const upcomingEligibility = sanction.festivals
+    .filter(
+      (item) =>
+        item.reservationEligibleAt &&
+        item.reservationEligibleAt.getTime() > Date.now(),
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/app/components/molecules/user-sanction-card.tsx:24:48
  22 |       (item) =>
  23 |         item.reservationEligibleAt &&
> 24 |         item.reservationEligibleAt.getTime() > Date.now(),
     |                                                ^^^^^^^^^^ Cannot call impure function
  25 |     )
  26 |     .sort(
  27 |       (a, b) =>

(react-hooks/purity)